### DISCUSSION TITLE
Search: search keyword only matches description

### DISCUSSION THREAD

**Discussion by chonki** [firefly-iii/firefly-iii#4304] [Firefly III] [answered]
Is it normal, that the search does not match the destination account or source account?

Exmample: When i search "Doctor" i get 0 results until i pass "destination_account_contains: Dcotor"

**Comment by JC5** [comment, marked as answer]
Indeed. You can also do `to:doctor` which is easier, but it was a decision to search in descriptions only for the time being.
